### PR TITLE
Feature/junit5

### PR DESCRIPTION
**Pull request by Anshika1603**
Please review my Junit5 assignment. As suggested earlier I have made changes in this assignment also.

### CODE REVIEW

File: src/main/java/com/knoldus/Person.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+    public void setAge(Integer age) {
+        if(age==0) {
+            this.age=age;
+            System.out.println("Error: Age cannot be zero");
+        }
+        else if (age<0) {
+            this.age=age;
+            System.out.println("Error: Invalid Age");
+        } else
```
Comment (anup0060) [inline src/main/java/com/knoldus/Person.java]: not good .. be consistent with brackets , even if one statement . 

else {

}

File: src/test/java/PersonTest.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        Assertions.assertEquals(0,person.getAge());
+        person.setAge(-76);
+        Assertions.assertEquals(-76,person.getAge());
+    }
+
+    @Test
+    public void testGetName_shouldReturnString_givenStringName() {
+        person.setName("Anshika");
+        Assertions.assertEquals("Anshika", person.getName());
+        person.setName("");
```
Comment (anup0060) [inline src/test/java/PersonTest.java:34]: empty case should be in a separate method

File: src/test/java/PersonTest.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        Assertions.assertEquals("Error! Empty Input.",person.getName());
+    }
+
+    @Test
+    public void testGetEmail_shouldReturnString_givenStringEmail() {
+        person.setEmail("anshika.srivastava@knoldus.com");
+        Assertions.assertEquals("anshika.srivastava@knoldus.com", person.getEmail());
+        person.setEmail("");
```
Comment (anup0060) [inline src/test/java/PersonTest.java:45]: same here

File: src/test/java/PersonTest.java [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+        Assertions.assertEquals("Error! Empty Input.",person.getEmail());
+    }
+
+    @Test
+    public void testGetAge_shouldReturnInteger_givenIntegerAge() {
+        person.setAge(22);
+        Assertions.assertEquals(22, person.getAge());
+        person.setAge(0);
+        Assertions.assertEquals(0,person.getAge());
```
Comment (anup0060) [inline src/test/java/PersonTest.java:54]: 0 - separate scenario

**Review by anup0060 — CHANGES_REQUESTED** [review]

**Comment by Anshika1603** [comment]
I have added separate methods for null cases and improved the brackets usage. Please review.

**Review by anup0060 — APPROVED** [review]

**Pull request merged**